evergreen.yml

```diff
@@ -141,18 +136,6 @@ tasks:
         params:
           command: 'c:\\python310\\python.exe -m venv ./venv'
           directory: curatorbin
-      - command: shell.exec
```

Comment:
Same thing as above, it seems like these were run on the latest waterfall commit.